src/guc.c

```diff
@@ -725,6 +736,22 @@ _guc_init(void)
 							   /* assign_hook= */ NULL,
 							   /* show_hook= */ NULL);
 
+	DefineCustomEnumVariable(/* name= */ "timescaledb.debug_require_vector_qual",
+							 /* short_desc= */
+							 "ensure that non-vectorized filters are used in DecompressChunk node",
```

Comment:
should this be vectorized instead of non-vectorized?

Reply:
It does both, depending on the setting. I'll make it say `vectorized or non-vectorized`.